recipes-ni/ni-hw-support/ni-hw-support-common_1.0.bb

```diff
@@ -0,0 +1,54 @@
+SUMMARY = "Support scripts and utilities common to all NI hardware products"
+DESCRIPTION = "Support scripts and utilities for all NI hardware products which are supported by NI LinuxRT."
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+LICENSE = "MIT"
+SECTION = "base"
+
+DEPENDS += "\
+	update-rc.d-native \
+"
+
+SRC_URI += "\
+	file://init.d/ni-rename-ifaces \
+"
+
+S = "${WORKDIR}"
+
+
+do_install () {
+	install -d ${D}${sysconfdir}/init.d/
+
+	install -m 0755 ${S}/init.d/ni-rename-ifaces ${D}${sysconfdir}/init.d/ni-rename-ifaces
+}
+
+pkg_postinst_${PN} () {
+	if [ -n "$D" ]; then
+		OPT="-r $D"
+	else
+		OPT="-s"
+	fi
+
+	update-rc.d $OPT ni-rename-ifaces start 38 S .
```

Comment:
Can we use the INITSCRIPT_NAME and INITSCRIPT_PARAMS provided by the update-rc.d class instead of open coding the update-rcd call here?

Reply:
I wish. But unless I'm blind and stupid, the [`update-rc.d.bbclass`](https://github.com/ni/openembedded-core/blob/nilrt/master/sumo/meta/classes/update-rc.d.bbclass) only supports a single initscript per package. Technically that would work for this PR, but I assume that this package (and others in the same family) will install multiple initscripts - where the only solution is to call update-rc.d yourself.